libquickevent/libquickeventcore/src/runstatus.cpp

```diff
@@ -43,113 +43,103 @@ bool RunStatus::isOk() const
 
 int RunStatus::toTime() const
 {
+	if (m_notCompeting)
```

Comment:
I am not sure if this is correct solution. If the runner is marked as NC and e.g. it is also set m_missingPunch flag, then he should have MissingPunch/MP/... status in results or NC?